src/utils.tsx

```diff
@@ -587,4 +587,4 @@ export const deselectRowById = <TData extends MRT_RowData>(
   });
 };
 
-export const booleanSelectFilterOptions = ['Yes', 'No'];
+export const COLUMN_FILTER_BOOLEAN_OPTIONS = ['Yes', 'No'];
```

Comment:
Could you put this next to the other column filter options above in the file, it makes it easier to see all of the similar values in one place.